types/node-insights/index.d.ts

```diff
@@ -35,7 +36,7 @@ declare class Insights {
      * Add insights data to the queue.
      * It is sent when the queue reaches a max size or a period of time has elapsed
      */
-    add(data: object, eventType?: string): void;
+    add(data: Record<string, any>, eventType?: string): void;
```

Comment:
The docs on npmjs.com seem to indicate that data can in fact be an arbitrary object and isn't restricted to `Record<string, any>`.